docs/en/docs/async.md

```diff
@@ -387,7 +387,7 @@ In previous versions of NodeJS / Browser JavaScript, you would have used "callba
 
 ## Coroutines
 
-**Coroutine** is just the very fancy term for the thing returned by an `async def` function. Python knows that it is something like a function that it can start and that it will end at some point, but that it might be paused ⏸ internally too, whenever there is an `await` inside of it.
+**Coroutine** is just the very fancy term for the thing returned by an `async def` function. Python knows that it is something like a function, that it can start, and that it will end at some point, but that it might be paused ⏸ internally too, whenever there is an `await` inside of it.
```

Comment:
```suggestion
**Coroutine** is just the very fancy term for the thing returned by an `async def` function. Python knows that it is something like a function, that it can start and that it will end at some point, but that it might be paused ⏸ internally too, whenever there is an `await` inside of it.
```

Reply:
Uh oh, we have an Oxford comma dissenter in our midst 😆 Totally works for me - it definitely makes the sentence flow better!